website/static/js/share/results.js

```diff
@@ -37,7 +38,11 @@ var Results = {
         };
 
         return m('', [
-            m('.row', m('.col-md-12', maybeResults(resultViews, vm.resultsLoading()))),
+            m('.row', m('.col-md-12', {
+                config: function(el, ini, ctx) {
+                    mathrender.typeset(el); // Adds all results to Mathjax queue at once.
```

Comment:
Does this need to be executed every time this view is rendered? Or can you just call this once on initialization, in which case you can use the second argument (what you're calling `ini`)?